subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/ScalaLibraryInitIntegrationTest.groovy

```diff
@@ -60,6 +61,7 @@ class ScalaLibraryInitIntegrationTest extends AbstractInitIntegrationSpec {
 
         and:
         commonJvmFilesGenerated(scriptDsl)
```

Comment:
Have you considered inlining `commonJvmFilesGenerated` with `mavenCentralRepositoryDeclared` as the repository is part of the generated files

Reply:
I started out that way and then thought that maybe this will look better when made explicit. But now I probably think it will be better to have it inlined. Will update.